src/main/scala/io/druid/indexer/spark/SparkBatchIndexTask.scala

```diff
@@ -68,7 +68,9 @@ class SparkBatchIndexTask(
   @JsonProperty("indexSpec")
   indexSpec: IndexSpec = new IndexSpec(),
   @JsonProperty("classpathPrefix")
-  classpathPrefix: String = null
+  classpathPrefix: String = null,
+  @JsonProperty("hadoopDependencyCoordinates")
+  hadoopDependencyCoordinates: util.List[String] = null
```

Comment:
any reason we use util.List here instead of scala Seq? Is this because we use the Druid objectMapper?